11/headful/al2022/Dockerfile

```diff
@@ -2,9 +2,18 @@ FROM amazonlinux:2022
 
 ARG version=11.0.17.8-1
 RUN set -eux \
-    && yum install -y https://corretto.aws/downloads/resources/$(echo $version | tr '-' '.')/java-11-amazon-corretto-headless-$version.amzn2022.$(uname -m).rpm https://corretto.aws/downloads/resources/$(echo $version | tr '-' '.')/java-11-amazon-corretto-$version.amzn2022.$(uname -m).rpm \
+    && rpm --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-amazon-linux-2022 \
+    && echo "localpkg_gpgcheck=1" >> /etc/dnf/dnf.conf \
+    && dnf install -y dnf-plugins-core \
+    && dnf download --downloaddir=/var/cache/dnf/corretto https://corretto.aws/downloads/resources/$(echo $version | tr '-' '.')/java-11-amazon-corretto-headless-$version.amzn2022.$(uname -m).rpm https://corretto.aws/downloads/resources/$(echo $version | tr '-' '.')/java-11-amazon-corretto-$version.amzn2022.$(uname -m).rpm \
```

Comment:
The base image already contains `curl`, right?  Is `dnf download` doing more than just downloading these to the specified directory?